Sign_Up_Demo/Extentions/UIAlertControllerExtention.swift

```diff
@@ -0,0 +1,8 @@
+import UIKit
+extension UIAlertController {
+    class func showAlertWith(viewController: UIViewController, message: String) {
```

Comment:
As this is part of every viewcontroller so you don't have to pass the viewcontroller instance and use self instead of viewController in this method.